outputs.tf

```diff
@@ -2,7 +2,24 @@
 # Outputs
 ########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "name" {
+  description = "Name of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.name
+}
+
+output "id" {
+  description = "ID of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.crn
```

Comment:
looks like this is the crn, so should the output name here not be `crn` ?

Reply:
crn and id are identical 
```
"crn" = "crn:v1:bluemix:public:sysdig-secure:eu-gb:a/abac0df06b644a9cabc6e44f55b3880e:1a4b73f9-fe78-4ae0-969e-279eccbc39be::"
  "id" = "crn:v1:bluemix:public:sysdig-secure:eu-gb:a/abac0df06b644a9cabc6e44f55b3880e:1a4b73f9-fe78-4ae0-969e-279eccbc39be::"
  ```
I will expose `crn` for now

Reply:
I think maybe add `id` too? As its available from `ibm_resource_instance.scc_wp.id`